default.build

```diff
@@ -170,9 +170,12 @@
   <target name="build_windows10" description="Build Windows 10 UAP" depends="clean">
     <if test="${os == 'Win32NT'}">
       <if test="${file::exists('c:\Program Files (x86)\MSBuild\Microsoft\WindowsXaml\v14.0\Microsoft.Windows.UI.Xaml.CSharp.targets')}">
-        <exec program="Protobuild" commandline="-generate WindowsUAP" />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU"' />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU"' />
+        <exec program="Protobuild" commandline="-generate WindowsUniversal" />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
```

Comment:
> 1) Wrong tool version.

This is detected based on the version of MSBuild you have installed.  If this is not 14.0, then you don't have 14.0 installed.  See https://github.com/hach-que/Protobuild/blob/master/Protobuild.Internal/BuildResources/GenerationFunctions.cs#L301.

> 2) Not including MonoGame.Framework.WindowsUAP.project.json

This was redundant with the Protobuild referencing.

> 3) Crazy assembly references:

Again, this is just doing what the MSBuild targets were doing internally anyway.  It's no different to the NuGet restore (except that this is explicit references instead of implied).

> 4) Also found this new thing as of this PR:

This is a new Protobuild feature that allows you to define post-build hooks for projects.  MonoGame doesn't use it, and it has no effect in this scenario (because you have no post-build hooks defined).